docs/advanced-features/s3-prefix-permissions.md

```diff
@@ -0,0 +1,34 @@
+# Restricting S3 access to specific prefix in bucket
+
+> User will still have access to the full lists of S3 objects,
+packages and logical keys inside of packages.
+
```

Comment:
Add the sentence:
"You can isolate user access to objects stored in specific S3 directories by defining an array of accessible `prefixes` in a custom IAM role or Amazon S3 bucket policy."